test/RootStateSchema.json

```diff
@@ -3064,6 +3064,9 @@
                                             },
                                             "quoteId": {
                                                 "type": "string"
+                                            },
+                                            "transferType": {
```

Comment:
Wasn't sure if this needed a migration. I don't think we need one because this is on a state field that is only relevant to the SelectProviderScreen and we refetch every time the screen is loaded with a new CICO flow. Also, even if we do a migration, don't think there's a good default we can set to since it can either be for cash in or cash out.

Reply:
that makes sense, i think it's fine to exclude the migration here :) 

i'm not close to the details but if there are flows where we always want to fetch new data and any persisted data from previous app sessions are not useful, we can consider excluding these from the redux persist? then we never need to worry about migrations. not sure if that's possible here though.

Reply:
cool, I ended up resetting the quotes field on the rehydrate action. I guess we can also stop persisting this field using something like https://github.com/rt2zz/redux-persist#nested-persists? I think this can be done in a follow up PR.